manifests/p/purocean/YankNote/3.3.7/purocean.YankNote.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: purocean.YankNote
+PackageVersion: 3.3.7
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64

```

```suggestion
- Architecture: x64
```
It looks like we need to start to move away from using `InstallerLocale` here because this prevents machines outside of the `en-US` locale from upgrading the application using the `winget upgrade` command.

See the comment that @denelon wrote at https://github.com/microsoft/winget-pkgs/issues/25347#issuecomment-903016232 & https://github.com/microsoft/winget-pkgs/issues/25347#issuecomment-903033816.